### DISCUSSION TITLE
Attibutes like ICommand, AlsoNotifyChangeFor and AlsoNotifyCanExecuteFor not found after update

### DISCUSSION THREAD

**Discussion by Phil4121** [CommunityToolkit/Maui#469] [Q&A] [answered]
Hello!

Recently I updated my VS 2022 Prev and all the packages in my Solution. I also did an update for the CommunityToolkit. Currently I'm using CommunityToolkit.Diagnostics. 8.0.0-preview4, CommunityToolkit.Maui 1.0.0 and CommunityToolkit.Mvvm 8.0.0-preview4.

Since the last update all the attributes I used like ICommand, AlsoNotifyChangeFor and AlsoNotifyCanExecuteFor are no longer recognized! The code generation for ICommand, etc. is not done anymore.

Anyone else has the same problem? 

Thanks, Phil

**Comment by VladislavAntonyuk** [comment, marked as answer]
https://github.com/CommunityToolkit/dotnet/releases/tag/v8.0.0-preview4

Please read release notes. There were breaking changes.

**Reply by Phil4121** [reply]
Thank you very much! Did not know that .Net Community Toolkit also has references to Maui Community Toolkit...

**Comment by CliffAgius** [comment]
Closed as answered
